### ISSUE TITLE
Update accessibility checklist vignette

### ISSUE THREAD

**Issue by matt-dray** [co-analysis/a11ytables#117]
The 'Tables' section of [Accessible spreadsheets: a checklist of the basics](https://analysisfunction.civilservice.gov.uk/policy-store/making-spreadsheets-accessible-a-brief-checklist-of-the-basics/) has the item 'Columns should be a sensible width', which is not in the [Accessibility guidance checklist](https://co-analysis.github.io/a11ytables/articles/checklist.html) vignette. The package currently supports a set column width and doubles it if the content is beyond a certain character length. Options for column width may be included as part of #77.
